app/controllers/application_controller.rb

```diff
@@ -15,14 +18,19 @@ def authenticate
   
     def authenticate_token
       authenticate_with_http_token do |token_key, options|
-        token_key = token_key.split(/(?:api_key=")(\w*)(?:"api_secret=")(\w*)/)
+        token_key = token_key.split(/(?:api_key=)(\w*)(?: api_secret=)(\w*)/)
```

Comment:
Is the space between api_key and api_secret intentional?

Reply:
yep.